### ISSUE TITLE
var not recognized in default java project

### ISSUE THREAD

**Issue by maxandersen** [redhat-developer/vscode-java#1211]
i have a java file using "var xyz = .." syntax and vscode tell me its an error despite my jvm is a java 11 version.

can vscode setup source target to latest version or is there a way to at least configure it ?

**Comment by snjeza** [comment]
I can't reproduce the issue. You can try set the following:
```
"java.home": "<your_jdk_11>"
```

**Comment by fbricon** [comment]
Works for me too
<img width="1102" alt="Screen Shot 2019-12-23 at 4 45 54 PM" src="https://user-images.githubusercontent.com/148698/71366966-ce581f00-25a3-11ea-8d0b-f64bf01ed5a5.png">

@maxandersen please attach your server log file (F1 > Java: Open Java language server log file)

**Comment by maxandersen** [comment]
okey, tried this again and it seems to work but it’s a bit weird.

turns out when I run code . from command line the environment was with java 8, but inside the terminal in vscode it was saying java 11....funny enough when I used sdkman to choose java 11 before launching then `var` worked but now in the vscode terminal I got java 8.......no idea why :)

but that is separate from this issue. this issue seem to work as long as user (like stupid me) make sure launch happens from env with java 11.
